lib/raxx.ex

```diff
@@ -964,8 +964,28 @@ defmodule Raxx do
   end
 
   @doc """
-  TODO
+  Helper function simplifying a list of `t:Raxx.part/0` parts.
+
+  Simplifying in this context means breaking down Request/Response objects
+  containing binary `body` values into their head, data and tail parts.
+
+  ## Examples
+      iex> response = response(:ok)
+      iex> response.body
+      false
+      iex> [response] == simplify_parts([response])
+      true
+
+      iex> response_with_body = response(:ok) |> set_body("some body")
+      iex> [head, data, tail] = simplify_parts([response_with_body])
```

Comment:
I wonder if `separate_parts` might be a better name. Not sure if people would immediately assume the list version was simpler

Reply:
I'm sold!

Reply:
I'm not as sold anymore (`break_down_parts` maybe?), but I changed it anyways :D